src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/BuildLogFailureReader.java

```diff
@@ -74,7 +73,7 @@ public FoundIndication scan(Run build) throws IOException {
             List<FoundFailureCause> foundFailureCauses = FailureReader.scanSingleLinePatterns(causes,
                                                                                               build,
                                                                                               reader,
-                                                                                              currentFile);
+                                                                                              "log");
```

Comment:
I can't remember what that file parameter is used for, a quick check in the current code this seems to be harmless. Maybe it's used for statistics or something?

Reply:
not sure, it was always called "log" when I checked though.

Reply:
maybe it should be a constant then?

Reply:
done